tests/rados/test_omap_entries.py

```diff
@@ -92,19 +103,29 @@ def run(ceph_cluster, **kw):
                     f"All the OSD's from the acting set {acting_set} were restarted "
                     f"and object movement completed for pool {pool_name}"
                 )
-        except Exception as e:
-            log.error(f"Failed with exception: {e.__doc__}")
-            log.exception(e)
-            return 1
-        finally:
-            log.info(
-                "\n \n ************** Execution of finally block begins here *************** \n \n"
+        # Verification of the Bug#2249003
+        client_node.exec_command(cmd=cmd_clearomap, sudo=True)
```

Comment:
Please capture the error string and validate it. 
``` [root@ceph-pdhiran-1-zxry5a-node7 ~]# rados clearomap -p test objectone
[root@ceph-pdhiran-1-zxry5a-node7 ~]# rados clearomap -p .mgr objectone
error clearing omap keys .mgr/objectone/(2) No such file or directory ```

Reply:
Captured the sting and displayed in the exception.